public/locales/en/gear.json

```
@@ -0,0 +1,3 @@
+{
+    "RefreshAvailability": "Roll for availability"
```

Since we have already rolled it for them at the start I think that it should be "Reroll".
```suggestion
    "RefreshAvailability": "Reroll availability"
```